app/src/main/res/values/strings.xml

```diff
@@ -13,4 +13,5 @@
     <string name="title_pressure">Pressure</string>
     <string name="title_current_detail">Current Detail</string>
     <string name="title_load_failure">Load data failure</string>
+    <string name="title_please_wait">Please wait...</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.title_please_wait` appears to be unused
 Unused resources make applications larger and slow down builds.